proto/minder/v1/minder.proto

```diff
@@ -1020,8 +1038,20 @@ message GetAuthorizationURLResponse {
 
 message StoreProviderTokenRequest {
     string provider = 1 [deprecated=true];
-    string access_token = 3;
-    optional string owner = 4;
+    string access_token = 3 [
+        (buf.validate.field).string = {
+            pattern: "^[A-Za-z0-9._-]$",
```

Comment:
I am not sure if this is sufficient. We should check the character set that GitHub, GitLab and DockerHub support for reference.

Reply:
Can you check that this regex is well-formed?  Often, you need to put `-` first in the character class to distinguish it from a character range.

Reply:
Oh, I see I missed the + sign actually so testing this `^[a-zA-Z0-9-_]+$` seems to work with gitlab/github tokens 
<img width="542" alt="image" src="https://github.com/user-attachments/assets/3d4e9f1c-251a-4ea3-9aad-28ab08deaaa3">

Reply:
Is that UI using RE2?

Reply:
Okay, it appears that the RE2 compiler doesn't find the string `0-9-_` ambiguous, so carry on!